packages/plugins/typescript/vue-apollo/src/visitor.ts

```diff
@@ -198,7 +198,7 @@ export class VueApolloVisitor extends ClientSideBaseVisitor<VueApolloRawPluginCo
       operationVariablesTypes,
     });
 
-    const compositionFunction = this.buildCompositionFunction({
+    let compositionFunction = this.buildCompositionFunction({
```

Comment:
```suggestion
    const compositionFunctions = [this.buildCompositionFunction({
```

prefer moving it as an array and use a `.join("\n")` in the return